python/GafferUI/VectorDataWidget.py

```diff
@@ -1043,6 +1037,57 @@ def getElement( self, rowIndex, columnIndex ) :
 _DataAccessor.registerType( IECore.V3fVectorData.staticTypeId(), _CompoundDataAccessor )
 _DataAccessor.registerType( IECore.V3dVectorData.staticTypeId(), _CompoundDataAccessor )
 
+class _ColorDataAccessor( _CompoundDataAccessor ) :
+
+	def __init__( self, data ) :
+
+		_CompoundDataAccessor.__init__( self, data )
+
+	def setElement( self, rowIndex, columnIndex, value ) :
+
+		if columnIndex < self.numColumns() - 1 :
+			_CompoundDataAccessor.setElement( self, rowIndex, columnIndex, value )
+		else :
+			self.data()[rowIndex] = GafferUI._Variant.fromVariant( value )
```

Comment:
This `_Variant` stuff was intended to deal with differences between PyQt and PySide, with PyQt exposing QVariant explicitly, and PySide doing implicit conversions to and from Python, so you never saw the QVariant type. I suspect this code wouldn't work with PyQt at all, because we have no conversion code for Color3f<->QVariant. I don't think that's necessarily a problem though since I think we're exclusively used with PySide now - it might be that the most pragmatic thing is to rip out the `_Variant` stuff completely (another time).